File: Algo201/problem_of_the_class/python/bee.py
def bee_each_year(n):
    if n == 0 :
        return 2
    if n == 1 :
        return 4
    return bee_each_year(n-2) + bee_each_year(n-1) + 1

print("input:")
n = int(input())
print("output")
print(bee_each_year(n))

File: Algo201/problem_of_the_class/java/Bee.java
import java.util.Scanner;
import java.util.ArrayList;
public class Bee{
    private static final Scanner in = new Scanner(System.in);
    private static ArrayList<Integer> beeEachYear(int n){
        ArrayList<Integer> pattern = new ArrayList<>();
        int init = 4; 
        int t1 = 3;
        int t2 = 5;
        pattern.add(init);
        while(n-- > 0){
            pattern.add(init += t1);
            int t3 = t1 + t2;
            t1 = t2;
            t2 = t3;
        } 
        return pattern;
    } 
    public static void main(String[] args) {
        int n = in.nextInt();
        System.out.println(beeEachYear(n).get(n-1));
    }
}